app/myfunctions.py

```diff
@@ -1,6 +1,6 @@
-def sortnumbers(num_list):
+def sort_numbers(num_list):
```

Comment:
It's extremely important to use idiomatic capitalization styles for function and class names. Function names are *always* written in `snake_case`.